### PR TITLE
Create VC Marketplace WorkItem Proposal

### PR DESCRIPTION
**Pull request by Gerstep**
Initial work item proposal for the VC marketplace subgroup.

### CODE REVIEW

File: work_items/vc_marketplace.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+By establishing the reference architecture for a Verifiable Credentials Marketplace, we would be able to discover, save, share and verify verifiable credentials in an anonymized manner across networks. This enables us to increase the usability and value of VCs across networks
+
+Consider the example of a migrant worker who doesn’t have the possibility to avail a loan due to missing Identification or credit score. With his income credentials generated from a particular employer he will be able to secure a loan from a different provider,
+
+(Mechanics of a real world credential exchange (protocol) covering value exchange can be defined to establish the reference architecture for a Verifiable Credentials Marketplace.)
+
+## Deliverables
+
+High level architecture for Verifiable Credentials marketplace independent from solution architecture / protocol design (open exchange) that covers
```
Comment (rado0x54) [inline work_items/vc_marketplace.md:22]: Sentence incomplete.

File: work_items/vc_marketplace.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+The group believes that the VC Marketplace can be separated into three distinct parts around Discovery, (Credential) Exchange, and Payments. In the first iteration the group will concentrate on the Discovery aspect.
+
+**Part 1 (Functional User story)**
```
Comment (rado0x54) [inline work_items/vc_marketplace.md:26]: Let's rename the Parts with the previously mentioned Titles (e.g. Discovery) and remove the first bullet point.

File: work_items/vc_marketplace.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+- Value exchange - separate scope/category
+- Tracking of VC sharing
+
+Part 3 (Out of scope for the first iteration)
+
+- Payments - combined scope with Value exchange
+
+What's the expected timeline?
+
+- 4 months (including a ramp-up in DEC)
```
Comment (rado0x54) [inline work_items/vc_marketplace.md:62]: Let's put an absolute timeline in here start-stop.

**Review by rado0x54 — CHANGES_REQUESTED** [review]
Some Minor comments. Also let's change the PR Title to "Workitem Proposal: VC Marketplace"

**Comment by Gerstep** [comment]
@rado0x54 thanks! 
Please, see changes reflected in the new commit.

File: work_items/vc_marketplace.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+High level architecture for Verifiable Credentials Marketplace independent from solution architecture / protocol design that covers the following items:
+
+- Use cases of VC Marketplace
+- VC metadata required to enable the Marketplace
+- Discovery mechanisms:
+    - Who are the issuers of credentials that I need?
+    - What credentials do I need to receive service?
+- Aggregated and anonymous tracking of VC Marketplace activity
+- Privacy, security, and compliance considerations across marketplace actors
```
Comment (wyc) [inline work_items/vc_marketplace.md:30]: Added this item, and if everyone is okay with the addition, then I'm fine to move this into the WG.

**Review by wyc — APPROVED** [review]

**Review by rado0x54 — APPROVED** [review]

File: work_items/vc_marketplace.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+**Part 3: Payments** (Out of scope for the first iteration)
+
+- Payments - combined scope with Value exchange
+
+Expected timeline
+
+- 4 months: January 2021 — April 2021
+
+Audience of the deliverables
```
Comment (rado0x54) [inline work_items/vc_marketplace.md:70]: @Gerstep Can we add the meeting time (weekly 8am-9am ET) here as well. (can also be done later)

File: work_items/vc_marketplace.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+(Mechanics of a real world credential exchange (protocol) covering value exchange can be defined to establish the reference architecture for a Verifiable Credentials Marketplace.)
+
+## Deliverables
+
+High level architecture for Verifiable Credentials Marketplace independent from solution architecture / protocol design that covers the following items:
+
+- Use cases of VC Marketplace
+- VC metadata required to enable the Marketplace
+- Discovery mechanisms:
```
Comment (glcohen) [inline work_items/vc_marketplace.md:26]: I'd love to see something about credential shapes -- how are schemas defined and consumed?

File: work_items/vc_marketplace.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+    - Definition of Credential Metadata
+    - Discovery of Interaction Partners by Credential Metadata
+- How can this metadata be captured?
+    - Leverage Credentials Manifest + Presentation Exchange and build upon
+    - Issuer Manifest — specifying types of VCs that can be requested
+    - Verifier Manifest — specifying types of VCs that are required to get services
+    - Credential Manifest — VC metadata
+- Some Example User Stories that the group will design:
+    - Issuer: How to communicate business and technical credential requirements to prospective holders?
+        - What Issuers do I accept?
```
Comment (glcohen) [inline work_items/vc_marketplace.md:46]: How do I check credential / revocation status?

File: work_items/vc_marketplace.md (hunk 5 above)
Comment (Gerstep) [reply work_items/vc_marketplace.md:74]: @rado0x54 I didn't get an invite. What day of the week are going with?

**Review by Gerstep — COMMENTED** [review]
Addressed comments by @glcohen

**Review by rado0x54 — APPROVED** [review]

**Pull request merged**